rpc_server.go

```diff
@@ -372,6 +374,9 @@ func (s *SourceControl) broadcastHeartbeat() {
 }
 
 func (s *SourceControl) broadcastStatus() {
+	if s.activeSource != nil {
```

Comment:
I'm not excited to have this in the STATUS object. It's fine to have it in the message to clients, but when stored in `config.yaml`, you have a problem: it's not a state that can become true just by loading the configuration file. How is that being handled?

Reply:
I agree with the principle that the yaml file ideally wouldn't contain things that can't be made True, but I don't think I should fix it here. This works fine, the line immediatley after loading the status from the yaml file is to `broadcastStatus` which overwrites `ChannelsWithProjectors`. There are other things that end up in the YAML file that fall into this catogory like `numberwritten`, `lancero.availablecards`, `countsseen`, `writing` (it could be made true, but it shouldn't be).